lib/BackbeatConsumer.js

```diff
@@ -241,6 +247,40 @@ class BackbeatConsumer extends EventEmitter {
         return this._consumer.once('ready', () => {
             this._consumerReady = true;
             this._checkIfReady();
+            if (this._logConsumerMetricsIntervalS !== undefined) {
+                this._consumer.on('event.stats', res => {
+                    const statsObj = safeJsonParse(res.message);
+                    if (statsObj.error) {
+                        this._log.error('error parsing consumer stats', { error: statsObj.error, topic: this._topic,
```

Comment:
Can we implement some tests to test the error paths of this function? As we test only the happy path, we should add tests to ensure we are confident when we get bad input objects (either the message or result.topic)

Reply:
I am not sure it is that simple to introduce.. neither if it has any interest, if we are not able to parse for some reason, we just don't log the lag offset..